gen-frontend.go

```diff
@@ -0,0 +1,5 @@
+package resource
```

Comment:
why not put all that in gen.go at the root of the project?

Reply:
issue created:
https://github.com/massalabs/thyra/issues/461